.github/actions/latest-stable-tag/get-latest-tag.js

```diff
@@ -0,0 +1,41 @@
+export function getLatestTag(tags) {
```

Comment:
It's only the latest stable tag, as you note in get-latest-tag.test.js. Should we be more explicit in the name?

Reply:
Yes, I will rename